utils/FactoryDeployerHelper.ts

```diff
@@ -1,8 +1,8 @@
 // SPDX-FileCopyrightText: 2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
 // SPDX-License-Identifier: Apache-2.0
 
-import { ContractFactory } from '@ethersproject/contracts';
-import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { SignerWithAddress } from '@nomicfoundation/hardhat-ethers/signers';
+import { ContractFactory } from 'ethers';
 import { deployments, ethers } from 'hardhat';
```

Comment:
```suggestion
import { deployments, ethers } from 'hardhat';
```
can we remove ethers dep here as we can get it from hardhat ?